src/build.d

```diff
@@ -845,7 +845,8 @@ auto getHostDMDPath(string hostDmd)
     {
         if (hostDmd.canFind("/", "\\"))
             return hostDmd;
-        return ["where", hostDmd].execute.output.lineSplitter.front;
+        return ["where", hostDmd].execute.output
+            .lineSplitter.filter!(file => file != srcDir.buildPath("dmd.exe")).front;
```

Comment:
I use a `dmd.bat` file.  I think you've broken my environment here.

Reply:
oh wait my mistake.  You're only filtering out `dmd.exe` in the current directory.  This is fine